rs/state_layout/src/state_layout.rs

```diff
@@ -48,6 +48,8 @@ mod tests;
 // State layout directory and file names.
 pub const CHECKPOINTS_DIR: &str = "checkpoints";
 pub const CANISTER_STATES_DIR: &str = "canister_states";
+pub const SNAPSHOTS_DIR: &str = "snapshots";
+pub const SNAPSHOT_FILE: &str = "snapshot.pbuf";
```

Comment:
The visualization of state layout from line 246 to 283 is not updated to the new layout.
(I miss gitlab where I can comment on any lines.)